mllm/nn/Functional.cpp

```
@@ -118,4 +118,13 @@ void scatter2Shards(const Tensor& src, const Tensor& shards_pointer, int32_t dim
                                            {src, shards_pointer});
 }
 
+Tensor scaledDotProductAttention(const Tensor& Q, const Tensor& K, const Tensor& V, const Tensor& mask) {
```

Clang (14.0.6)

[error] 4 adjacent parameters of 'scaledDotProductAttention' of similar type ('const int &') are easily swapped by mistake

(bugprone-easily-swappable-parameters,-warnings-as-errors)